CHANGELOG.rst

```diff
@@ -1,3 +1,16 @@
+2.1.0 (2024-02-21)
```

Comment:
Actually we set the date during the release process. 👍 

```suggestion
2.1.0 (UNRELEASED)
```

Reply:
Applied, thanks!